robot_controllers/include/robot_controllers/cartesian_twist.h

```diff
@@ -125,10 +127,11 @@ class CartesianTwistController : public Controller
   virtual std::vector<std::string> getClaimedNames();
 
   /** @brief Controller command. */
-  void command(const geometry_msgs::Twist::ConstPtr& goal);
+  void command(const geometry_msgs::TwistStamped::ConstPtr& goal);
 
 private:
   KDL::Frame getPose();
+  KDL::Frame cartPose;
```

Comment:
cartPose is only used in one place -- inside the update() -- therefore, it should be defined in update (also, the variable name should be cart_pose in that callback)